scripts/common.py

```diff
@@ -266,6 +266,23 @@ def install_npm_library(library_name, version, path):
             'yarn', 'add', '%s@%s' % (library_name, version)])
 
 
+def ask_user_to_confirm(message):
+    """Asks user to perform a task and confirm once they are done.
+
+    Args:
+        message: str. The message which specifies the task user has
+            to do.
+    """
+    while True:
+        python_utils.PRINT(
+            '******************************************************')
+        python_utils.PRINT(message)
+        python_utils.PRINT('Confirm once you are done by entering y/ye/yes.\n')
```

Comment:
Hmm, why is `ye` also included?

Reply:
I would recommend excluding it.

Reply:
As background, I think I originally wrote this snippet of code. When I wrote it originally, I included 'ye' to be more forgiving since it's unlikely that someone who types that means anything other than "yes". But I'm happy either way.

Reply:
I think it is fine to have both `ye` and `yes`. @kevinlee12 is that ok?

Reply:
That's fine if it was already there.